msg_helpers.go

```diff
@@ -275,7 +275,7 @@ func unpackString(msg []byte, off int) (string, int, error) {
 		case b == '"' || b == '\\':
 			s.WriteByte('\\')
 			s.WriteByte(b)
-		case b < 32 || b > 127: // unprintable
+		case b < ' ' || b > '~': // unprintable
```

Comment:
I think using integer values is slightly better, as this signals "This is ASCII",not some Go unicode

Reply:
inconsistency for the win ! let's leave the chars here then as well.